include/envoy/upstream/outlier_detection.h

```diff
@@ -99,5 +99,9 @@ class Detector {
 
 typedef std::shared_ptr<Detector> DetectorPtr;
 
+struct SRAccumulatorBucket {
```

Comment:
I would either move this struct into the class where you are using it or add a comment about what its intents are.